### PR TITLE
ton of issues

### LINKED ISSUE #4070: Contact Form Submission - Suggestion (Silver - Binary Search)
Someone submitted the contact form!

**URL**: https://usaco.guide/silver/binary-search?lang=cpp
**Module**: Silver - Binary Search
**Topic**: Suggestion
**Message**: 
Unusual use of notation in quiz question 1. Big O and Big Omega shouldn't be conflated with "worst-case" and "best-case", respectively. Describing the best-case runtime as Omega(1) is correct but doesn't mean much. It would be better to use theta or big O notation.

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

should close #4070 i hope

### CODE REVIEW

File: content/3_Silver/Binary_Search.mdx [hunk 1]
```diff
@@ -1198,27 +1198,26 @@ def binary_search(x: int, a: list):
 </PySection>
 </LanguageSection>
 
-
     <Quiz.Answer>
-      $\mathcal{O}(\log n)$ / $\Omega(\log n)$
+      $\mathcal{O}(\log n)$ / $\mathcal{O}(\log n)$
```
Comment (bqi343) [inline content/3_Silver/Binary_Search.mdx:1202]: This should use $\Theta$ rather than $\mathcal O$ for all quiz answers

Comment (SansPapyrus683) [reply content/3_Silver/Binary_Search.mdx]: alright

File: content/6_Advanced/String_Search.problems.json [hunk 2]
```diff
@@ -262,6 +262,18 @@
         "year": 2008
       }
     },
+    {
+      "uniqueId": "cf-888G",
```
Comment (envyaims) [inline content/6_Advanced/String_Search.problems.json:266]: string search???
trie isnt just for strings yknow

Comment (bqi343) [reply content/6_Advanced/String_Search.problems.json:266]: it is, right? integers are binary strings

Comment (SansPapyrus683) [reply content/6_Advanced/String_Search.problems.json:266]: benq said to put it here

Comment (envyaims) [reply content/6_Advanced/String_Search.problems.json:266]: mmmmmm ok

**Review by envyaims — APPROVED** [review]

**Pull request merged**